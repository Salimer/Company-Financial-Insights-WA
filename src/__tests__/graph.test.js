import React from 'react';
import { render } from '@testing-library/react';
import Graph from '../components/graph';

test('renders the graph correctly', () => {
  const company = [
    { date: '2021-01-01', revenue: 1000000 },
    { date: '2022-01-01', revenue: 1500000 },
    { date: '2023-01-01', revenue: 2000000 },
  ];

  const { container } = render(<Graph company={company} />);
  expect(container.firstChild).toMatchSnapshot();
});
