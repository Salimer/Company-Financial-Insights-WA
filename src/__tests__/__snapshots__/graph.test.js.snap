// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the graph correctly 1`] = `
<section
  class="sc-bgqQcB ePBDDx bg-headlineBg flex flex-col items-center"
>
  <h1
    class="text-white text-lg p-5 md:p-10"
  >
    Revenue for the past 5 years
  </h1>
  <div
    class="p-3 w-full"
  >
    <canvas
      height="150"
      role="img"
      width="300"
    />
  </div>
</section>
`;
